httpc.go

```diff
@@ -203,9 +203,14 @@ func (r *Request) AcceptedResponseCodes(acceptedResponseCodes []int) *Request {
 
 // Run executes a request
 func (r *Request) Run() error {
+	return r.RunWithContext(context.Background())
```

Comment:
Is this fully backwards compatible in terms of timeout behavior?

Reply:
The tests pass, so there doesn't seem to be a difference, which also is my experience with the `Foo` and `FooWithContext` pattern from other libs and my own work.

see: https://github.com/fako1024/httpc/pull/3/commits/9796ef4c0b0aadc842bbc8eea464b11825d30f2d#diff-eece9a4e1f4133dbfc7a30b3ca3e8c6b5ba397d0409aafa20348821b00cb18caL296

on the `Run()` path, it just moves the old base context used for the timeout up the call stack.